podaac/subsetter/subset.py

```diff
@@ -699,8 +709,27 @@ def build_cond(str_timestamp, compare):
                 timestamp = np.datetime64(timestamp) - epoch_datetime
 
         if np.issubdtype(dataset[time_var_name].dtype, np.dtype(float)):
-            start_date = np.datetime64(dataset[time_var_name].attrs['Units'][-10:])
-            timestamp = (np.datetime64(timestamp) - start_date).astype('timedelta64[s]').astype('float')
+            start_date = None
```

Comment:
I think I see what you're doing here now. I would prefer to have this logic not in this function and instead do this before we get to this point. At this point in the code, we should be able to assume that the time var we are operating on consists of absolute times as some kind of datetime object.